### ISSUE TITLE
Large parskip and itemize

### ISSUE THREAD

**Issue by CarlAndersson** [FrankMittelbach/fmitex#11]
Using a large skip in the parskip package will produce unexpected results in itemize environments. I would expect the space between items in a list to be unaffected by the chosen paragraph spacing. This seems to be one of the ideas with the package at least.

```
\documentclass[12pt,final,a4paper]{article}

\usepackage{lipsum}
\usepackage[skip=2cm]{parskip}


\begin{document}
\lipsum[1]

\lipsum[1]

\begin{itemize}
    \item Foo
    \item Bar
    \item Baz
\end{itemize}
\end{document}
```

**Comment by CarlAndersson** [comment]
I can add that it seems to interact with titles as well.

**Comment by FrankMittelbach** [comment]
@CarlAndersson the package is and remains a hack, I only tried to do a little improvement to what was in existence before. As a result it behaves halfway reasonable for common use cases with small but noticable `\parskip` values, but I agree that stops being the case if you do something like `skip=2cm` and I should probably put a note in to that effect.

But it is not that it would make sense to fully make something like lists ignore a loose typesetting, e.g. in your case with `skip=2cm` you would get a `\parsep` (the space between paragraphs in a list item) of 2cm. If then the `\itemsep` would be untouched that would look ridicuous.

In short, for a proper solution (involving such a case) instead of using that package one needs a full design with all kind of values being adjusted, someting that this package doesn't and in fact can't handle.

And even if it could, given that it was out there for 20 odd years with many people actually using it it couldn't strong alter its behavior. So bottom line, I have to call this works as designed originally. I leave it open for now because, as I said, I should add some notes to that affect.
